subprojects/testing-jvm/src/integTest/resources/org/gradle/testing/junit/JUnitJdkNavigationTest/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply plugin: 'java'
+
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile 'junit:junit:4.12'
+    testCompile 'org.apache.fexli:org.osgi.foundation:1.2.0'
```

Comment:
I think you have a typo here in the dependency `groupId`. Should be `org.apache.felix`. Did you try to run the test?